isaactest/tests/pwd_reset_throttle.py

```diff
@@ -46,9 +47,9 @@ def pwd_reset_throttle(driver, Users, ISAAC_WEB, WAIT_DUR, **kwargs):
                     wait_for_invisible_xpath(driver, "//div[@class='toast-message']/h4", 0.5)
                 except TimeoutException:
                     raise TimeoutException("Password reset error message unexpectedly shown after %s requests!" % password_resets)
-                time.sleep(0.5)
-                message = driver.find_element_by_xpath("(//p[@ng-show='passwordResetFlag'])[2]")
-                assert "Your password request is being processed." in message.text
+                time.sleep(2)
+                message = driver.find_element_by_xpath("(//*[@ng-show='passwordResetFlag'])")
+                assert 'Your password reset request is being processed. Please check your inbox.' in message.get_attribute("innerHTML")
```

Comment:
`.text` does not work - `p` and `strong` don't work either, `*` does, it could be I have a different webdriver version but I wouldn't have thought that would affect it much